build.xml

```diff
@@ -155,7 +155,6 @@ java -cp ${jar.absolute.path}:${lib.classpath} kbasesearchengine.tools.SearchToo
         <sysproperty key="test.cfg" value="${test.cfg}"/>
         <batchtest>
           <fileset dir="${test.src}">
-            <include name="kbasesearchengine/test/parse/**Test.java"/>
```

Comment:
There were 2 tests covered by this line but only one was added below, so a test is missing

Reply:
There are two, split by arfath's comment

Reply:
ObjectParserTests was already there. A test is missing: https://github.com/kbase/KBaseSearchEngine/tree/develop/test/src/kbasesearchengine/test/parse

Reply:
Done